x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/get_signals_template.ts

```diff
@@ -7,6 +7,7 @@
 
 import signalsMapping from './signals_mapping.json';
 import ecsMapping from './ecs_mapping.json';
+import indicatorMapping from './indicator_mapping.json';
```

Comment:
Since this file is just currently "ECS event mappings," what do you think about using `ecsMapping.mappings.properties.event` instead of this new file? I expect we'll want these indicator mappings to be in sync with the latest from ECS, so that might be the best way to accomplish this.